capnp.opam

```diff
@@ -6,24 +6,22 @@ bug-reports: "https://github.com/pelzlpj/capnp-ocaml/issues"
 dev-repo: "https://github.com/pelzlpj/capnp-ocaml.git"
 author: "Paul Pelzl <pelzlpj@gmail.com>"
 maintainer: "Paul Pelzl <pelzlpj@gmail.com>"
-build: [["env" "PREFIX=%{prefix}%" "omake"]]
-build-test: [
-  ["env" "PREFIX=%{prefix}%" "omake" "src/tests/run-tests"]
-  [ocaml "./src/tests/run-tests.run"]
-]
-install: [["env" "PREFIX=%{prefix}%" "omake" "install"]]
-remove: [["env" "PREFIX=%{prefix}%" "omake" "uninstall"]]
+
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+build-test: ["jbuilder" "build" "@runtest" "@src/benchmark/benchmarks"]
+
 depends: [
-  "omake"
+  "jbuilder" {build}
   "ocamlfind" {>= "1.5.1"}
```

Comment:
It's no longer necessary to have an `ocamlfind` dependency once you switch to jbuilder, as it pulls it in if it needs it.